src/components/ui/SuggestsDropdown.jsx

```diff
@@ -3,6 +3,13 @@ import classnames from 'classnames';
 import { object, func, string, arrayOf, bool } from 'prop-types';
 
 import SuggestItem from './SuggestItem';
+import { isMobileDevice } from '../../libs/device';
+
+const blurField = () => {
+  if (isMobileDevice()) {
```

Comment:
Can you use the `DeviceContext` instead ?

Reply:
yes, but shouldn't we remove isMobileDevice everywhere?